README.md

```diff
@@ -22,6 +22,29 @@ A user should be able to interact with the echo server as follows:
 ## Setup
 1. [Clone the repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) to your local computer.
 2. Use your terminal to navigate into your new http_server folder and install project dependencies by running `mix compile`.
-3. To start the Echo Server, call `mix start` from within the http_server folder.
-4. To connect to the Server as a client, [install telnet](https://formulae.brew.sh/formula/telnet) and call `telnet 127.0.0.1 4040` in a local terminal.
-5. To run the tests, call `mix test` from within the http_server folder.
+3. You may notice that the `test/http_server_spec` submodule exists, but has no files. Call `git submodule init` and `git submodule update` from within the http_server folder to initialize your local git modules configuration file and fetch all data from the [http_server_spec](https://github.com/8thlight/http_server_spec) project.
+4. To start the Echo Server, call `mix start` from within the http_server folder.
+5. To connect to the Server as a client, [install telnet](https://formulae.brew.sh/formula/telnet) and call `telnet 127.0.0.1 4040` in a local terminal.
+
+## Testing
+### Running the ExUnit Test Suite:
+To run the ExUnit test suite, call `mix test` from within the http_server folder.
+
+### Acceptance Tests:
+
+#### Dependencies
+* Ruby 2.7.6
+
+### Running the Acceptance Test Suite:
+1. Start your HTTP server by calling `mix start` from within the http_server folder.
```

Comment:
On this branch, `mix start` is not working for me. (It does work on the `main` branch).
Here is the error I am seeing:
```shell
$ mix start
** (MatchError) no match of right hand side value: {:error, :eaddrinuse}
    (http_server 0.1.0) lib/http_server.ex:6: HTTPServer.accept/1
    (mix 1.14.2) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.2) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```

Reply:
<img width="499" alt="Screenshot 2023-05-30 at 2 42 29 PM" src="https://github.com/kelseyroy/http_server/assets/89801075/94eecbee-d7ae-4ca6-8b52-d55fcad88365">

If you go to System Settings -> General -> Airdrop & Handoff, is the Airplay Receiver on or off? This is a problem I ran into on Friday because the acceptance tests run on port 5000, which is also the port that Airplay uses. If it's on, turning it off will probably fix the problem or I can change the port both the server and acceptances tests use!

Reply:
My Airplay Receiver is unchecked and it is working for me, so that might be the problem? The only error I see is when it does not "gracefully" exit (but that's work for another ticket) 😄

Reply:
Would it be easier to just update the acceptance tests to use a the defaut elixir port, since this is an elixir project? vs. asking everyone to change local settings?

https://github.com/kelseyroy/http_server_spec/blob/e8ee68f121327edfd0f55b46f9208af64910f2c7/server.rb#L1-L7